library/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -30,6 +35,22 @@ public object Chucker {
             .setFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
     }
 
+    public fun Context.createShortcut() {
```

Comment:
This function should follow structure of other functions in this object. `@JvmStatic` and accept context as an argument.

If you make it public it should also have a documentation. Moreover leaving it public requires mocked function in `no-op` library.

Reply:
I made it as a internal function.